hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueService.kt

```diff
@@ -23,7 +23,7 @@ class MissingTopicException(message: String) : RuntimeException(message)
 
 open class HmppsQueueService(
   private val telemetryClient: TelemetryClient?,
-  hmppsTopicService: HmppsTopicService,
+  hmppsTopicFactory: HmppsTopicFactory,
```

Comment:
this is reverting back to how it was before the standard and async versions.  With the topic service I couldn't seem to get the queue factory wired up before the health indicator stuff.